### PR TITLE
Add WIN & QT build scripts

### PR DESCRIPTION
**Pull request by OHerasym**
### PR Changes: ###

1. Add WIN platform build & run tests script
2. Add QT platform build & run tests script

Related: [APPLINK-25165](https://adc.luxoft.com/jira/browse/APPLINK-25165)

### CODE REVIEW

**Comment by OHerasym** [comment]
@anosach-luxoft , @dev-gh , Please review

File: make_test_native.bat [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+call "setup_VS2013_x64.cmd"
+
+cd build/sdl_win_x64
+
+call "c:\Program Files (x86)\MSBuild\12.0\Bin\MSBuild.exe" smartDeviceLinkCore.sln /t:Build /p:Configuration=Debug
```
Comment (dev-gh) [inline make_test_native.bat]: @OHerasym 
Plz, use environment variables for that path

File: make_test_qt.cmd [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+call "setup_Qt_x86.cmd"
+
+cd build/sdl_win_qt_x86
+call nmake
```
Comment (dev-gh) [inline make_test_qt.cmd]: I would suggest to use jom instead.

**Comment by dev-gh** [comment]
Reviewed

File: make_test_native.bat (hunk 1 above)
Comment (OHerasym) [inline make_test_native.bat]: @dev-gh , 09b40c1, Done

File: make_test_qt.cmd (hunk 2 above)
Comment (OHerasym) [inline make_test_qt.cmd]: @dev-gh , 09b40c1, Done

**Comment by dev-gh** [comment]
Reviewed

**Pull request merged**